cwl/src/main/scala/wdl4s/cwl/AddEmbeddedCwl.scala

```diff
@@ -0,0 +1,50 @@
+package wdl4s.cwl
+
+import shapeless.{Path => _, _}
+import ammonite.ops._
+import cats.syntax.traverse._
+import cats.syntax.either._
+import cats.instances.list._
+import cats.effect.IO
+import CwlDecoder.{Parse, ParseValidated}
+import cats.{Applicative, Monad}
+import cats.data.EitherT._
+import cats.data.EitherT
+import lenthall.validation.ErrorOr._
+
+object AddEmbeddedCwl extends Poly1 {
+  val workflowStepLens = lens[Workflow].steps
+  val workflowStepRunLens = lens[WorkflowStep].run
+
+  implicit def workflow =
```

Comment:
Exactly :).  `Result` type is being assigned within the body of this specific `Case`, and yeah TIL you can declare types w/ blocks